src/strings/Strings.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace strings;
+
+class Strings implements StringsInterface
+{
+
+    public function snakeCaseToCamelCase(string $input): string
+    {
+        $arrayInput = explode('_', $input);
+        $camelCase = lcfirst($arrayInput[0]);
+
+        foreach ($arrayInput as $key => $string) {
+            if ($key != 0) {

```

Comment:
Please avoid using the "if" in each iteration. You can remove the first item from `$arrayInput` before `foreach`